Converted  InheritedKVs.vue component to TS #12088

Anuragsharma15-dell opened the pull request:
Closes https://github.com/kestra-io/kestra/issues/12088.

Converted InheritedKVs.vue to use the Composition API with TypeScript for better maintainability and type safety.

Commit:
refactor(ui): migrate FlowEdit.vue to Composition API + TypeScript

Commit:
refactor(ui): migrate Labelinput.vue to Composition API + TypeScript

Commit:
Added the InheritedKVs.vue component to typescript

Anuragsharma15-dell committed:
Merge branch 'develop' into feat/InheritedKVS-api

MilosPaunovic reviewed (changes requested):
There are 3 files changed in this PR, and it needs to be only one.

MilosPaunovic commented:
Just a small note for the future, if you could use **GitHub Keywords** when opening PRs to connect it to the proper issue (to automate the entire process a bit more), it would be super helpful to us. You can read a bit more about it [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue) (I've added closing keyword for this particular issue).

Anuragsharma15-dell committed:
Merge branch 'develop' into feat/InheritedKVS-api

Anuragsharma15-dell commented:
Ah sorry for the inconvinience sir as i am only 19 and form not a good place trying to learn new things would love to work with your guidance

Anuragsharma15-dell commented:
I have also deleted the unnecessary branches

MilosPaunovic commented:
For me to review this PR, you'd need to revert the changes in the 2 unnecessary files.

Anuragsharma15-dell committed:
Delete ui/src/components/flows/FlowEdit.vue

Anuragsharma15-dell committed:
Delete ui/src/components/labels/LabelInput.vue

Anuragsharma15-dell committed:
Merge branch 'develop' into feat/InheritedKVS-api

Anuragsharma15-dell commented:
sir i have reverted the change from the 2 unnecessary files but i am not able to remove them from ui and added only the Ihnherited.vue component to typescript please reveiew my pr or if  there  any issue tell me

MilosPaunovic commented:
You don't need to delete the whole content of the file, just to revert your changes.
